src/plugins/action_trim_responses/src/databaseMethods.ts

```diff
@@ -64,6 +66,38 @@ const databaseMethods = (DBConnect: any) => ({
       throw err
     }
   },
+  updateApplicationResponseTimestamps: async (responsesToUpdate: number[], timestamp: string) => {
+    const text = `UPDATE application_response
+      SET time_updated = $1
+      WHERE id = ANY ($2)
+      RETURNING (SELECT code FROM
+        template_element WHERE id = template_element_id)
+      `
+    try {
+      const result = await DBConnect.query({ text, values: [timestamp, responsesToUpdate] })
+      return result.rows.map((row: { code: string }) => row.code)
+    } catch (err) {
+      console.log(err.message)
+      throw err
+    }
+  },
+  updateReviewResponseTimestamps: async (responsesToUpdate: number[], timestamp: string) => {
+    const text = `UPDATE review_response
+      SET time_updated = $1
+      WHERE id = ANY ($2)
+      RETURNING (SELECT code FROM
```

Comment:
hmm why do we need 'code' from application_response ? (maybe from template_element ?)

Reply:
It's `code` from application_response JOIN template_element -- so it is the template_element code ;)